### ISSUE TITLE
pry gem has incompatible dependency

### ISSUE THREAD

**Issue by trevorthornton** [harvard-library/aspace-import-excel#18]
pry is specified at '~> 0.10.4', which is dependent on method_source ~> 0.8.1, but AS frontend Gemfile specifies method_source 0.9.0.

Here is the error:

```
Bundler could not find compatible versions for gem "method_source": In snapshot (Gemfile.lock): method_source (= 0.9.0) In Gemfile: pry (~> 0.10.4) java was resolved to 0.10.4, which depends on method_source (~> 0.8.1) java rails (= 5.0.1) java was resolved to 5.0.1, which depends on railties (= 5.0.1) java was resolved to 5.0.1, which depends on method_source
...
```

**Comment by bobbi-SMR** [comment]
>sigh<  I'm assuming this is for 2.2.2, which we haven't installed yet, so I didn't catch the error.  Leaving Pry in the Gemfile unrestrained breaks everything *before* 2.2.2.

I'll have to figure out the best way of handling this.

**Comment by trevorthornton** [comment]
Yes, 2.2.2. Sorry. Would it be feasible to add a branch for 2.2.2 for now?

**Comment by bobbi-SMR** [comment]
If I can't get a quick fix that solves for all, I'll do that.  btw, since Gemfile.lock is no longer part of the repo, you could also feel free in just removing the second parameter to gem, and do the scripts/initialize-plugin.sh in the meantime :-)

**Comment by bobbi-SMR** [comment]
... and **these** are the kinds of issues that have to get into the plugin documentation.  Good thing I'm on the case 1/2 :-)

**Comment by trevorthornton** [comment]
I removed the version parameter for pry in the Gemfile and initialized it under 2.2.2 and it works!

**Comment by bobbi-SMR** [comment]
Great!

Since we don't have 2.2.2 installed yet, would you let me know what version the Gemfile.lock has for pry?

**Comment by trevorthornton** [comment]
0.11.3-java. Here's the whole Gemfile.lock for reference:

```
GEM
  remote: http://rubygems.org/
  specs:
    coderay (1.1.2)
    ffi (1.9.18-java)
    method_source (0.9.0)
    nokogiri (1.8.1-java)
    pry (0.11.3-java)
      coderay (~> 1.1.0)
      method_source (~> 0.9.0)
      spoon (~> 0.0)
    rubyXL (3.1.0)
      nokogiri (>= 1.4.4)
      rubyzip (>= 1.0.0)
    rubyzip (1.2.1)
    spoon (0.0.6)
      ffi
```

**Comment by bobbi-SMR** [comment]
Thanks; I'm going to have to write a (potentially hairy) conditional in the gem file to account for the change in Aspace versions, but it will give me something to write about in the documentation :-)

**Comment by trevorthornton** [comment]
Implemented the new version on AS 2.2.2 and the issue seems to be resolved - thanks!
